lib/url.js

```diff
@@ -36,7 +36,7 @@ const {
   URLSearchParams,
   domainToASCII,
   domainToUnicode,
-  formatSymbol
+  formatSymbol,
```

Comment:
Unnecessary comma

Reply:
@starkwang I'll fix it

Reply:
Trailing commas are fine and if we didn't have so much code that lacked them, we would probably even have a lint rule to enforce them. Anything that cleans up git diffs in the long-term is a positive.